workflows/scrnaseq.nf

```diff
@@ -133,21 +133,34 @@ workflow SCRNASEQ {
         ch_txp2gene = KALLISTO_BUSTOOLS.out.txp2gene
     }
 
-    // Run salmon alevin pipeline
-    if (params.aligner == "alevin") {
-        SCRNASEQ_ALEVIN(
+    // Run simpleaf pipeline
+    if (params.aligner == "simpleaf" || params.aligner == "alevin") {
+
+        SCRNASEQ_SIMPLEAF(
             ch_genome_fasta,
             ch_filter_gtf,
             ch_transcript_fasta,
-            ch_salmon_index,
+            ch_simpleaf_index,
             ch_txp2gene,
             ch_barcode_whitelist,
             protocol_config['protocol'],
-            ch_fastq
+            params.simpleaf_umi_resolution,
+            ch_fastq,
+            [] // for existing map dir; not applicable
         )
-        ch_versions = ch_versions.mix(SCRNASEQ_ALEVIN.out.ch_versions)
-        ch_multiqc_files = ch_multiqc_files.mix(SCRNASEQ_ALEVIN.out.alevin_results.map{ meta, it -> it })
-        ch_mtx_matrices = ch_mtx_matrices.mix( SCRNASEQ_ALEVIN.out.alevin_results )
+        ch_versions = ch_versions.mix(SCRNASEQ_SIMPLEAF.out.ch_versions)
+        ch_multiqc_files = ch_multiqc_files.mix(SCRNASEQ_SIMPLEAF.out.quant.map{ meta, it -> it })
+        ch_mtx_matrices = ch_mtx_matrices.mix(
+            SCRNASEQ_SIMPLEAF.out.quant.map{
+                meta, files -> [
+                    meta +
+                    [input_type: meta["filtered"] ? "filtered" : "raw" ],
```

Comment:
based on the docs, the `simpleaf` ones would be treated as `raw`, why the conditional is needed? Is something coming as `filtered`?

Reply:
I see. Super.
Can you also update in the docs [here](https://github.com/DongzeHE/scrnaseq/blob/dev/docs/output.md#other-output-data) regarding this? That alevin-fry results are not always with suffix "raw"?

Reply:
Super. Thanks for the detailed explanation. Don't think is needed for now.